mako/filters.py

```diff
@@ -161,3 +161,5 @@ def htmlentityreplace_errors(ex):
     "str": "str",
     "n": "n",
 }
+
+CONFLICT_PREFIX = "__ALIAS_"
```

Comment:
I think this would be better as

    DEFAULT_ESCAPE_PREFIX = "__DEFAULT_ESCAPE_"